src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -98,14 +92,15 @@ const useGroupInfo = () => {
       const sendData = {
         user_id: selectedUser.user_id
       };
+      const members = [];
+      members.push(profile?.myProfile?.user_id, selectedUser?.user_id);
       const processGetBlock = await checkUserBlock(sendData);
-
       if (!processGetBlock.data.data.blocked && !processGetBlock.data.data.blocker) {
-        return openChatMessage();
+        return createSignChat(members, selectedUser);
       }
       return handleOpenProfile(selectedUser);
     } catch (e) {
-      console.log(e, 'eman');
+      console.log(e, 'erro');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's an error handling issue here. The catch block only logs the error but doesn't handle it in any meaningful way. It would be better to provide some feedback to the user or at least report this error to your error tracking system.
```diff
- console.log(e, 'erro');
+ // replace with your error reporting tool
+ reportError(e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->